src/utils/work/fetchWork.js

```diff
@@ -117,7 +121,7 @@ const fetchWork = async (
   // If caching is disabled, we add an additional randomized key to the hash so we never reuse
   // past results.
   let cacheUniquenessKey = null;
-  if (environment !== Environment.PRODUCTION && localStorage.getItem('disableCache') === true) {
+  if (environment !== Environment.PRODUCTION && localStorage.getItem('disableCache') === 'true') {
```

Comment:
will line `localStorage.getItem('disableCache')` fail if `disableCache` doesn't exist in localStorage?

Reply:
No, if it doesn't exist, it'll return `null`